docs/examples/tools/cloud_resource_detector/resource_detector_trace.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opentelemetry import trace
+from opentelemetry.exporter.cloud_trace import CloudTraceSpanExporter
+from opentelemetry.sdk.resources import get_aggregated_resources
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import SimpleExportSpanProcessor
+from opentelemetry.tools.resource_detector import GoogleCloudResourceDetector
+
+# MUST be run on a Google tool!
+# Detect resources from the environment
+resources = get_aggregated_resources([GoogleCloudResourceDetector()])
```

Comment:
same comment

Reply:
I don't believe so, since its required that this code be run in a Google tool. The only case where raise_on_error should be used is code could be run both on AWS and GCE, meaning both GoogleCloudResourceDetector and AWSResourceDetector would be present (and obviously one failing). Otherwise, the user should just not attempt detection.